ecl2df/vfp/_vfp.py

```diff
@@ -506,9 +511,9 @@ def vfp_main(args) -> None:
             table_number = int(
                 vfp_table.schema.metadata[b"TABLE_NUMBER"].decode("utf-8")
             )
-            vfp_filename = f"{outputfile}_{str(table_number)}.arrow"
+            args["output"] = f"{outputfile}_{str(table_number)}.arrow"
```

Comment:
Confusing for later readers that things are added to the variables (?) outside the scope of giving variables?